doc/orchestrator.rst

```diff
@@ -165,3 +165,511 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrator Example
+=============================
+SmartSim offers functionality to automate the deployment of multiple
+databases, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+In the following sections, we simulate a simple version of this use case.
+
+The example is comprised of two script files:
+
+* The :ref:`Application Script<The Application Script>`
+* The :ref:`Experiment Script<The Experiment Script>`
+
+**The Application Script demonstrates:**
+
+1. Connecting SmartRedis clients within the application to retrieve tensors
+   from the standard databases to store in a colocated database.
+
+**The Experiment Script demonstrates:**
+
+1. Launching two standard Orchestrators with unique identifiers.
+2. Creating a colocated model (launches one colocated Orchestrator) with
```

Comment:
To address @juliaputko's point below. One thing that you should make clear is that the Experiment driver script launches the application script with a co-located database. You kind of mention it, but it's unclear in the current wording what the relationship between the two is.

Reply:
I was struggling with that, thank you!